README.md

```diff
@@ -1 +1,156 @@
-# angular-directive-controllers-readme
\ No newline at end of file
+# Directive Controllers
+
+## Overview
+
+Whilst using `scope` is awesome, we can't really do much with it other than just display some data that is passed through to us. What if we need to manipulate this data? Well, this is where our controllers come in.
+
+## Objectives
+
+- Describe Directive Controllers
+- Create a Controller using "controller"
+- Import an existing Controller to a Directive
+
+## Directive Controllers
+
+We can create controllers for our directives - awesome!
```

Comment:
Add some more context here - we have data, but we're still not harnessing all of what Angular can offer.  To do this, we need a controller directly for our directives...